src/interface/src/app/treatments/stand-data-chart/stand-data-chart.component.ts

```diff
@@ -0,0 +1,148 @@
+import { Component } from '@angular/core';
+import { ChartConfiguration } from 'chart.js';
+import { NgChartsModule } from 'ng2-charts';
+import { AsyncPipe, JsonPipe, NgIf } from '@angular/common';
+import { DirectImpactsStateService } from '../direct-impacts.state.service';
+import { map, Observable } from 'rxjs';
+import { SLOT_COLORS, YEAR_INTERVAL_PROPERTY } from '../metrics';
+import { filter } from 'rxjs/operators';
+import { MapGeoJSONFeature } from 'maplibre-gl';
+
+const baseFont = {
+  family: 'Public Sans',
+  size: 14,
+  style: 'normal',
+  weight: '600',
+};
+
+@Component({
+  selector: 'app-stand-data-chart',
+  standalone: true,
+  imports: [NgChartsModule, NgIf, AsyncPipe, JsonPipe],
+  templateUrl: './stand-data-chart.component.html',
+  styleUrl: './stand-data-chart.component.scss',
+})
+export class StandDataChartComponent {
+  constructor(private directImpactsStateService: DirectImpactsStateService) {}
+
+  activeStandValues$: Observable<number[]> =
+    this.directImpactsStateService.activeStand$.pipe(
+      filter((s): s is MapGeoJSONFeature => s !== null),
+      map((a) => {
+        return Object.values(YEAR_INTERVAL_PROPERTY).map(
+          (property) => a.properties[property] * 100
+        );
+      })
+    );
+
+  barChartData$ = this.activeStandValues$.pipe(
+    map((data) => {
+      return {
+        labels: [0, 5, 10, 15, 20],
+        datasets: [
+          {
+            data: data,
+            barThickness: 30,
+          },
+        ],
+      } as ChartConfiguration<'bar'>['data'];
+    })
+  );
+
+  private readonly staticBarChartOptions: ChartConfiguration<'bar'>['options'] =
```

Comment:
I guess I'll need to catch up on how we do chart.js configs, but sometime in the future, maybe we can coordinate on a way to organize these so we're blending less styles into ts code.

Reply:
yep I think a step two is moving some of this stuff to a constant...somewhere. Some sort of base options config for both charts